code/controllers/subsystem/atoms.dm

```diff
@@ -159,13 +162,12 @@ SUBSYSTEM_DEF(atoms)
 		if(location)
 			/// Sends a signal that the new atom `src`, has been created at `loc`
 			SEND_SIGNAL(location, COMSIG_ATOM_AFTER_SUCCESSFUL_INITIALIZED_ON, A, arguments[1])
```

Comment:
This is another 500ms of just init time alone (InitAtom is still extremely hot post-init) so we should see if we can work to remove it too

Reply:
Blegh, looks like this is old and was just a rename that I didn't catch